Packs/Box/Integrations/BoxV2/BoxV2.yml

```diff
@@ -3132,7 +3132,7 @@ script:
     - contextPath: Box.Folder.item_status
       description: The status of the parent of the item.
       type: String
-  dockerimage: demisto/pyjwt3:1.0.0.13142
+  dockerimage: demisto/pyjwt3:1.0.0.16907
```

Comment:
We have an open issue regarding the docker image. - https://github.com/demisto/etc/issues/33818

Reply:
Done